src/xdocs/index.xml.vm

```diff
@@ -124,6 +124,13 @@
       </p>
     </section>
 
+    <section name="Limitations">
+      <p>
+        Checkstyle is a single file static analysis tool, for more details please read
+        <a href="writingchecks.html#Limitations">full list</a>.
```

Comment:
`full list` => `the full list of limitations`
I think only the word `limitations` should be linked.